src/utils.jl

```diff
@@ -16,18 +16,25 @@ function get_scalar_term(model::MOI.ModelLike,
                                                            S <: MOI.AbstractScalarSet}
     
     if F <: SVF
-        [- MOIU.getconstant(get_set(model, ci))]
+        return [- MOIU.getconstant(get_set(model, ci))]
     else
-        MOIU.constant(get_function(model, ci)) .- MOIU.getconstant(get_set(model, ci))
+        return MOIU.constant(get_function(model, ci)) .- MOIU.getconstant(get_set(model, ci))
     end
 end
 
 function get_scalar_term(model::MOI.ModelLike, 
```

Comment:
You should be able to replace this function by `MOI.constant` with MOI v0.9

Reply:
That's a good question. I am pretty convinced that it will work when I change to MOI 0.9 but should I start developing using MOI in master or I can change everything once MOI 0.9 is released?